benchmarks/src/main/java/io/druid/benchmark/query/GroupByProfile.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.benchmark.query;
+
+//import com.yourkit.api.Controller;
+
+import org.openjdk.jmh.infra.Blackhole;
+
+/**
+ * Test app for profiling GroupByBenchmark behavior. Uses the proprietary YourKit API, so this file
+ * is commented out to prevent dependency resolution issues. Kept here for future usage/reference.
+ */
+public class GroupByProfile
```

Comment:
I took out these *Profile classes, that seems simplest at this point.

The sections had to be commented out since they depend on a yjp-controller-api-redist.jar which doesn't have an official version from yourkit in maven (there are a few third-party copies of it though, e.g.: http://mvnrepository.com/artifact/com.facebook.thirdparty.yourkit-api/yjp-controller-api-redist)

I saw some jmh/yourkit-related project on github but it hasn't been updated since 2014